dashboard/src/shared/Auth.test.ts

```diff
@@ -58,29 +59,42 @@ describe("Auth", () => {
     });
   });
 
-  describe("fetchOIDCToken", () => {
-    it("should fetch a token", async () => {
-      Axios.head = jest.fn(path => {
-        expect(path).toEqual("");
-        return { headers: { authorization: "Bearer foo" } };
+  describe("isAuthenticatedWithCookie", () => {
+    it("returns true if request to API root succeeds", async () => {
+      Axios.get = jest.fn(path => {
+        return { headers: { status: 200 } };
```

Comment:
nit: this should probably return a promise too (to match the other tests)

Reply:
Updated.